src/main/scala/zio/package.scala

```diff
@@ -307,6 +308,9 @@ package object zmx extends MetricsDataModel with MetricsConfigDataModel {
           } yield longs
         }
 
+      // polls a metric from the ring buffer
```

Comment:
Can you clean up these comments?

Reply:
I removed the added comments on `poll` and `collect`. I kept the added "TODO" comment on the `untilNCollected` schedule and the return value of the `listen` method, though.